WTWSMS/common/landed_titles/WtWSMS_tribes.txt

```diff
@@ -952,13 +952,19 @@ d_goths = {
 	# tribe = yes
 	
 	allow = {
-		always = no
+		OR = {
+			culture = gothic
+			culture = crimean_gothic
+			culture = ostrogothic
+		}
+		controls = 560
+		prestige = 500
```

Comment:
@loup99 

Understood. I thought "always = no" meant it could never be formed. I understand what it means, now. I'll add it back.